src/adjoint/DASolver/DASolver.C

```diff
@@ -533,19 +531,27 @@ void DASolver::getForcesInternal(
     volumeForceField.write();
 
     // The above volumeForceField is face-centered, we need to interpolate it to point-centered
-    label pointCounter = 0;
-    List<label> globalIndex = pointList.clone();
-    globalIndex = -1;
-
     pointField meshPoints = meshPtr_->points();
 
     vector nodeForce(vector::zero);
 
+    label patchStart = 0;
     forAll(patchListSort, cI)
     {
         // get the patch id label
         label patchI = meshPtr_->boundaryMesh().findPatchID(patchListSort[cI]);
-
+        label patchIPoints = boundaryMesh.findPatchID(patchListSort[cI]);
```

Comment:
Will these two IDs be the same for all cases? Is it unnecessary to get the patch ID from both the mesh and point mesh boundaries?